### ISSUE TITLE
compute code coverage

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#1420]
We should look into tracking coverage of our testsuite.

Does this just involve using ``DEAL_II_SETUP_COVERAGE`` and running the testsuite with ``CTEST_COVERAGE()``?

**Comment by bangerth** [comment]
I think that is what it would be for deal.II. There is a paragraph in the description of the deal.II testsuite that may help to figure out how to do it.

**Comment by tjhei** [comment]
Yes, see https://www.dealii.org/developer/developers/testsuite.html#coverage

But this enables the compiler flags to generate coverage information (basically ``-ftest-coverage``) also for when we compile ASPECT.

**Comment by ljhwang** [comment]
Currently is it running Coverage? I do not see it as a separate section.

Best,
-Lorraine

*****************************
Lorraine Hwang, Ph.D.
Associate Director, CIG
530.752.3656

**Comment by tjhei** [comment]
> Currently is it running Coverage? I do not see it as a separate section.

I am not sure what are you asking about. There is currently no public coverage output for you to see for deal.II or ASPECT, but we have support for this in deal.II.

**Comment by ljhwang** [comment]
That answered the question!

Best,
-Lorraine

*****************************
Lorraine Hwang, Ph.D.
Associate Director, CIG
530.752.3656
